code/modules/RCD/schematics/tile.dm

```diff
@@ -9,6 +9,26 @@
 
 #define PAINT_ASK_DESC   1
 
+/datum/rcd_schematic/clear_decals
+	name = "Clear Decals"
+	category = "Painting"
+
+	flags = RCD_GET_TURF
+
+/datum/rcd_schematic/clear_decals/attack(var/atom/A, var/mob/user)
+	to_chat(user, "Clearing decals...")
+	playsound(get_turf(master), 'sound/effects/spray3.ogg', 15, 1)
+	if (!do_after(user, A, 20))
+		return 1
+
+	playsound(get_turf(master), 'sound/machines/click.ogg', 50, 1)
+
+	var/turf/T = A
+	if (!isturf(T))
```

Comment:
I don't actually know why I even put this check in because it's impossible. The schematic has the flag `RCD_GET_TURF` so clicking on an object calls it for the turf anyways.